### ISSUE TITLE
Getting constant output from Serial Monitor in DEBUG mode

### ISSUE THREAD

**Issue by mechachoi** [ShikyC/Sanro-Arduino#18]
Hello ShikyC,

Thank you so much for developing this. I am very passionate about getting my own taiko setup up and running. 

I have my Taiko (an official arcade taiko) connected to my circuit with bridge rectifiers. I am seeing approximately 4V constant voltage out of the output of the rectifiers when I put my multimeter across it, but I do see a change when I strike the drum in the respective locations.

However, whenever I open the serial monitor to see what is happening in DEBUG mode, I am getting basically constant values from the Serial Monitor, 126945 just over and over again. I'm trying to figure out what's going on that would cause this behavior. 

If I change one of the sensitivities of one of the sections of the drum, all the values change from 126945 but still they do not react to the pounding of the drum.

**Comment by ShikyC** [comment]
The constant 4V output from the rectifier seems erronous - it should output 0V if you don't hit the drum. Unfortunately I don't have an official drum so I can't say if it works with my circuits (I only tested with Taiko Force Lv. 5 and Lv. 6).

You can debug this issue with the following steps (preferable with an oscilloscope - but a sensitive multimeter may also work):

1. Disconnect everything, directly measure the output of the drum.
   - It should be 0V when the drum is not hit, and about -5V to +5V when the drum is hit.
2. Using only the rectifier (don't connect to the microcontroller or the circuit board yet), measure the output of the rectifier.
   - It should be 0V when the drum is not hit, and 0V to +5V when the drum is hit.
3. Assemble the circuit, connect to the microcontroller, power on, and measure the output of the rectifier.
   - It should be identical to the results in step 2.
 
Following the steps, please check when this problem happens so I can further help you.

**Comment by mechachoi** [comment]
Doesn't the piezo drum need some kind of powered circuit to output signal?

**Comment by ShikyC** [comment]
No. Unlike active input sources (such as microphones), piezoelectric sensors produces periodic voltage by themselves as if they're power sources.
